doc/source/admin/interactive_environments.rst

```diff
@@ -74,20 +75,38 @@ Running ``node lib/main.js --help`` should produce some useful help text
     --verbose
 
 There are two ways to handle actually running the proxy. The first is to have
-Galaxy automatically launch the proxy as needed, the second is to manage it
-with something like Supervisord. The command for launching the proxy manually
-(or via supervisord) is
+Galaxy automatically launch the proxy as needed. This is the default configuration
+as of 2014. Alternately, the proxy can be stated manually or via a system such as
+Supervisord. The command for launching the proxy is:
 
 .. code-block::  console
 
     $ node $GALAXY_ROOT/lib/galaxy/web/proxy/js/lib/main.js --ip 0.0.0.0 \
         --port 8800 --sessions $GALAXY_ROOT/database/session_map.sqlite \
         --cookie galaxysession --verbose
 
+And this can be configured in your supervisord config by adding:
+
+.. code-block:: console
+
+
+    [program:galaxy_nodejs_proxy]
+    directory       = GALAXY_ROOT
+    command         = GALAXY_ROOT/lib/galaxy/web/proxy/js/lib/main.js --sessions database/session_map.sqlite --ip 0.0.0.0 --port 8800
+    autostart       = true
+    autorestart     = unexpected
+    user            = GALAXY_USER
+    startsecs       = 5
+    redirect_stderr = true
+
+where GALAXY_ROOT is the location of your Galaxy installation and GALAXY_USER is the username of the user that
```

Comment:
GALAXY_ROOT -> `''GALAXY_ROOT''` (except instead of ', use `, which I can't seem to get to escape properly, sorry)